More infection tweaks

comma opened the pull request:
Cuts infection rate of burns. Should be less 100% germ increase on every tick now
Also fixes some weird check that resulted in wounds /always/ increasing organ's germ level, even if they were not infected. Now wound needs to get infected first, and chance per tick to increase germ levele is capped at 30%

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

ghost commented:
Can you make really minor wounds not get infected? Because it's extremely annoying sometimes.

comma commented:
They aren't supposed to, anything below 10 damage (per wound) is supposed to be safe, derped check just increased germ value of organ as along as ANY wounds existed, small, big, infected, treated, anything.

Ithalan commented:
Would you consider also making the spaceacilin checks related to infection check for spaceacilin in the stomach in addition to the bloodstream? Stomach content gets metabolized directly rather than going to the bloodstream now as far as I can tell, so spaceacilin pills are rather useless for fighting infections right now.

comma commented:
Could be kinda awkward but doable I guess.

comma commented:
Done

comma committed:
More infection tweaks

Cuts infection rate of burns. Should be less 100% germ increase on every tick now
Also fixes some weird check that resulted in wounds /always/ increasing organ's germ level, even if they were not infected. Now wound needs to get infected first, and chance per tick to increase germ levele is capped at 30%
